CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ### Added
 
+- [#4045](https://github.com/firecracker-microvm/firecracker/pull/4045)
+  Added `snapshot-editor` tool for modifications of snapshot files.
```

Comment:
I think we can go a little bit more into details here. What exactly can it modify? Also, "snapshot files" is a bit ambiguous here, since it could refer to both the memory and the vmstate file. We should clarify that is only allows modification of the vmstate file.

Reply:
Maybe it can also link to the documentation you add in this PR?